.goreleaser.yml

```diff
@@ -1,8 +1,50 @@
 builds:
-  - binary: sidecar
+  - binary: spiffe-helper
     goos:
       - darwin
       - linux
     goarch:
       - amd64
 
+nfpms:
```

Comment:
goreleaser was remove from actual build, can you try to use another plugin in order to generate rpms?

Reply:
Sure, I'll update this PR to build the RPM in a docker container.